config/webpack.config.js

```diff
@@ -23,6 +23,7 @@ const getClientEnvironment = require('./env');
 const ModuleNotFoundPlugin = require('react-dev-utils/ModuleNotFoundPlugin');
 const ForkTsCheckerWebpackPlugin = require('react-dev-utils/ForkTsCheckerWebpackPlugin');
 const typescriptFormatter = require('react-dev-utils/typescriptFormatter');
+const ServiceWorkerWebpackPlugin = require('serviceworker-webpack-plugin');
```

Comment:
Da się też zrobić bez plugina ;) Wystarczy stworzyć nowy entrypoint dla webpacka, ale będziesz musiał wtedy troszkę rzeczy zrobić ręcznie. Teraz większość ogarnia Ci ten plugin.
Póki co zostawmy - jak stiwerdzisz, że już nie masz co robić to pokombinujesz bez pluginu - dzięki czemu zrozumiesz, jak to działa pod spodem.